freeipa.spec.in

```diff
@@ -540,6 +541,7 @@ Requires: initscripts
 %endif
 Requires: libcurl >= 7.21.7-2
 Requires: xmlrpc-c >= 1.27.4
+Requires: jansson >= 2.12
```

Comment:
Is there a reason you are requiring 2.12? Does it provide some minimum level for bug fixing or is it just the current release?

Reply:
It's the current release available in F32 and Debian Buster. The [changelog](https://jansson.readthedocs.io/en/2.13/changes.html) mentioned some bugfixes for `json_pack`, although I didn't take a detailed look.
No reason otherwise.

Reply:
Constraint removed.